theme/less/components/dropdown.less

```diff
@@ -1,84 +0,0 @@
-/*
----
-name: Dropdown
-category: Components
----
-
-## Dropdown
-Dropdowns are used throughout the website, as selectors, or as togglers.
-Their construction is based on a standard secondary button, but with some specifics (arrow inserted after the text, supports an `.active` class...)
-You an use a `<select>` element that will be reset to display correctly on all browsers, or any arbitrary HTML element with the `.label` and `.select` classes.
-
-A: without label (options are the label)
-
-B: without options (only specified label is shown)
-
-For interactivity, please check the corresponding section of this styleguide.
-
-```dropdown-select.html
-<div class="dropdown">
-    <select>
-        <option>Option A</option>
-        <option>Option B</option>
-    </select>
-</div>
-```
-*/
-
-.dropdown {
```

Comment:
We can remove the `search.html` file but we'll have to remove the entire topic directory because `search.html` is used inside `subnav.html` and this one is included in the `display_base.html` extended in all other files.